libs/sdk-ui-kit/src/Header/HeaderInviteButton.tsx

```diff
@@ -0,0 +1,16 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import { useIntl } from "react-intl";
+import { IHeaderInviteButtonProps } from "./typings";
+
+export const HeaderInviteButton: React.FC<IHeaderInviteButtonProps> = ({ onInviteButtonClick }) => {
+    const intl = useIntl();
+    return (
+        <div className="gd-invite-button" onClick={onInviteButtonClick}>
+            <i className="gd-icon-invite" />
```

Comment:
Please rename `gd-icon-invite` → `gd-header-invite-icon`,  to be consistent with the naming for account icon at https://github.com/gooddata/gooddata-ui-sdk/blob/468fd09e48a7267216aef93c04f2ecde6006d0d9/libs/sdk-ui-kit/styles/scss/header.scss#L477